src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const BooksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books = state.books.filter((book) => book.id !== action.payload);
+    },
+  },
+});
+
+export const { addBook, removeBook } = BooksSlice.actions;
+export default BooksSlice.reducer;
```

Comment:
- [ ] I highly recommend you change the name of this file to `categoriesSlice` and not BooksSlice.  
- [ ] For this file, you're expected to have one method `checkStatus` that returns the text `Under construction` and not these reducers. Kindly remove these reducers and create 1 reducer - checkStatus in this file as required 👍🏽
- [ ] Let's export the checkStatus reducer after you implement it and the actions as well 👍🏽